common/culture/traditions/00_regional_traditions.txt

```diff
@@ -1107,93 +1108,122 @@ tradition_desert_nomads = {
-		# 0 = intrigue
-		# 1 = mediterranean
-		# 4 = crown.dds
-	# }
+	layers = {
+		0 = intrigue
+		1 = mediterranean
+		4 = crown.dds
+	}
 	
-	# is_shown = {
-		# any_parent_culture_or_above = {
-			# OR = {
-				# this = culture:nubian
-				# this = culture:daju
-			# }
-		# }
-	# }
-	# can_pick = {
-		# any_parent_culture_or_above = {
-			# OR = {
-				# this = culture:nubian
-				# this = culture:daju
-			# }
-		# }
-	# }
+	is_shown = {
+		OR = {
+			has_cultural_pillar = heritage_darnassian
+			has_cultural_pillar = heritage_avianic
+			this = culture:shivarra
+			this = culture:hyldnir
+			this = culture:blood_troll
+			this = culture:mantid
+		}
+	}
+	
+	can_pick = {
+		custom_tooltip = {
+			text = female_preference_law_faith_or_culture_trigger
+			
+			scope:character = {
+				scope:character = { is_male = no }
+				primary_title = {
+					NOR = {
+						has_title_law = male_only_law
+						has_title_law = male_preference_law
+					}
+				}
+				NOR = {
+					has_realm_law = male_only_law
+					has_realm_law = male_preference_law
+				}
+			}
+		}
+	}
 
-	# parameters = {
-		# gender_equal_inheritance = yes
-		# female_only_inheritance = yes
-		# female_preference_inheritance = yes
-		# prowess_from_martial_education = yes
-	# }
+	parameters = {
+		gender_equal_inheritance = yes
+		female_only_inheritance = yes
+		female_preference_inheritance = yes
+		prowess_from_martial_education = yes
+	}
 	
-	# cost = {
-		# prestige = {
-			# add = {
-				# value = tradition_base_cost
-				# desc = BASE
-				# format = "BASE_VALUE_FORMAT"
-			# }
-			# if = {
-				# limit = {
-					# NOT = {
-						# any_in_list = { list = traits this = flag:ethos_egalitarian }
-					# }
-				# }
-				# add = {
-					# value = tradition_incompatible_ethos_penalty
-					# desc = not_egalitarian_desc
-				# }
-			# }
-			# if = {
-				# limit = {
-					# trigger_if = {
-						# limit = {
-							# scope:character = {
-								# is_ai = no
-							# }
-						# }
-						# NAND = {
-							# any_ruler = {
-								# count >= 3
-								# culture = prev
-								# primary_title.tier > tier_barony
-								# is_male = yes
-							# }
-							# any_ruler = {
-								# count >= 3
-								# culture = prev
-								# primary_title.tier > tier_barony
-								# is_female = yes
-							# }
-						# }
-					# }
-					# trigger_else = {
-						# always = yes
-					# }
-				# }
-				# add = {
-					# value = tradition_unfulfilled_criteria_penalty
-					# desc = three_gender_rulers_desc
-				# }
-			# }
+	cost = {
+		prestige = {
+			add = {
+				value = tradition_base_cost
+				desc = BASE
+				format = "BASE_VALUE_FORMAT"
+			}
+			if = {
+				limit = {
+					any_in_list = { list = traits this = flag:ethos_courtly }
+					any_in_list = { list = traits this = flag:ethos_egalitarian }
+					any_in_list = { list = traits this = flag:ethos_bureaucratic }
+				}
+				add = {
+					value = tradition_incompatible_ethos_penalty
+					desc = not_bellicose_stoic_or_communal_desc
```

Comment:
Desc is different from the triggers.

Reply:
This is intentional. The trigger adds a penalty for the courtly, egalitarian and bureaucratic ethoses.

Reply:
But the desc mentions bellicose.

Reply:
It's a negative desc. (Defined in Vanilla CK3 `cultural_traditions_I_english.yml`)

```
not_bellicose_stoic_or_communal_desc:1 "The [culture|E] does not have the $ethos_bellicose_name$, $ethos_stoic_name$ or $ethos_communal_name$ [ethos|E]"
```

Reply:
The desc mentions other ethoses, not the ones you used in `limit`.